noxfile.py

```diff
@@ -32,6 +32,14 @@ def install_package(package_name: str, package_version: str, session) -> None:
         raise
 
 
+def get_component_test_tempfile_prefix(name: str):
```

Comment:
@therightstuff I don't understand why this indirection of temp files

Reply:
as discussed out of band, it makes it easier to understand which temp files belong to which tests, i thought that might be useful (and it's a little cleaner imo, and it only cost me a couple of minutes)